hudi-client/pom.xml

```diff
@@ -231,6 +231,13 @@
       <artifactId>hive-exec</artifactId>
       <version>${hive.version}</version>
       <scope>test</scope>
```

Comment:
but this was `test` scoped right.. so the issues you ran into were around running tests using spark 2.4.4?

Reply:
Yes, the issues we faced was during running on tests.  @modi95 please confirm

Reply:
Yes - this is a fix to an issue that shows up only while running unit tests.